compiler/src/typed/btype.re

```diff
@@ -408,27 +408,30 @@ let forget_abbrev = (mem, path) =>
 /*  Utilities for labels          */
 /**********************************/
 
+module Parsetree = Grain_parsing.Parsetree;
```

Comment:
This should not be using a Parsetree type. We alias the Asttypes and then use the alias in the `typed/` directory